satellite/marketing/marketingweb/server.go

```diff
@@ -186,11 +190,23 @@ func formToStruct(w http.ResponseWriter, req *http.Request) (o marketing.NewOffe
 	return o,nil
 }
 
+func (s *Server) reRenderTables(w http.ResponseWriter){
+    req,err := http.NewRequest("GET","/",nil)
```

Comment:
This `NewRequest` shouldn't be here, it doesn't make sense to make a new request and then redirect. Redirect the original request.